Makefile

```diff
@@ -67,7 +67,7 @@ phpvendors:
 	$(MAKE) -C $(FOSRCDIR) phpvendors
 
 preparetest: ./utils/prepare-test
```

Comment:
`make preparetest` is still breaking the system without warning. I would suggest to remove that target and call `./utils/prepare-test -y -a` directly in the travis script